UPGRADE-4.0.md

```diff
@@ -55,6 +75,8 @@ FrameworkBundle
     - `"form.type.submit"`
     - `"form.type.reset"`
 
+   `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed
```

Comment:
Why did you add this line here ? it should be form section and it misses the `*`